virtual-machine/openstack.yaml

```diff
@@ -109,7 +105,7 @@ node_templates:
         image_id: { get_input: image }
         flavor_id: { get_input: flavor }
         user_data: { get_attribute: [ cloud_init, cloud_config ] }
-      use_external_resource: { get_input: use_existing_server }
+      use_external_resource: false
```

Comment:
@bartoszkosciug `false` is the default value of this property so you can just delete the line.